### PR TITLE
AntiMicroX.antimicrox version 3.3.3

### LINKED ISSUE #95669: [Package Request]: AntiMicroX
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not a script file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: AntiMicroX
Publisher Url: https://github.com/AntiMicroX/antimicrox
Package Name: AntiMicroX
Description: Graphical program used to map keyboard buttons and mouse controls to a gamepad.
Package Version: 3.3.3
Installer URL: https://github.com/AntiMicroX/antimicrox/releases/tag/3.3.3

Current Package Identifier (for new versions): N/A

### PR DESCRIPTION
**Pull request by afermar**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
Resolves #95669

### CODE REVIEW

File: manifests/a/AntiMicroX/antimicrox/3.3.3/AntiMicroX.antimicrox.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: AntiMicroX.antimicrox
+PackageVersion: 3.3.3
+PackageLocale: en-US
+Publisher: AntiMicroX
+PackageName: AntiMicroX
```
Comment (stephengillie) [inline manifests/a/AntiMicroX/antimicrox/3.3.3/AntiMicroX.antimicrox.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://github.com/AntiMicroX
PublisherSupportUrl: https://github.com/AntiMicroX/antimicrox/issues
PackageName: AntiMicroX
PackageUrl: https://github.com/AntiMicroX/antimicrox
License: GPL v3
LicenseUrl: https://github.com/AntiMicroX/antimicrox/blob/master/LICENSE
```

File: manifests/a/AntiMicroX/antimicrox/3.3.3/AntiMicroX.antimicrox.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+PackageVersion: 3.3.3
+PackageLocale: en-US
+Publisher: AntiMicroX
+PublisherUrl: https://github.com/AntiMicroX
+PublisherSupportUrl: https://github.com/AntiMicroX/antimicrox/issues
+PackageName: AntiMicroX
+PackageUrl: https://github.com/AntiMicroX/antimicrox
+License: GPL v3
+LicenseUrl: https://github.com/AntiMicroX/antimicrox/blob/master/LICENSE
+License: GPL v3
```
Comment (stephengillie) [inline manifests/a/AntiMicroX/antimicrox/3.3.3/AntiMicroX.antimicrox.locale.en-US.yaml:14]:
```suggestion
```

Comment (stephengillie) [reply manifests/a/AntiMicroX/antimicrox/3.3.3/AntiMicroX.antimicrox.locale.en-US.yaml:14]: Apologies, I missed that the license was already added.

File: manifests/a/AntiMicroX/antimicrox/3.3.3/AntiMicroX.antimicrox.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: AntiMicroX.antimicrox
+PackageVersion: 3.3.3
+PackageLocale: en-US
+Publisher: AntiMicroX
```
Comment (Trenly) [inline manifests/a/AntiMicroX/antimicrox/3.3.3/AntiMicroX.antimicrox.locale.en-US.yaml:7]:
```suggestion
Publisher: Humanity
```
This should match Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]
```
--> Comparing ARP Entries

DisplayName DisplayVersion Publisher ProductCode
----------- -------------- --------- -----------
AntiMicroX  3.3.3          Humanity  AntiMicroX
```

**Review by Trenly — APPROVED** [review]

**Pull request merged**